hw4/hw4.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+
+if __name__ == "__main__":
+    arr0 = np.zeros(10)
+    arr1000_7 = np.array([i for i in range(1000, 0, -7)])

```

Comment:
Делать список, а затем превращать его в эррей - неэффективно. Если это можно создать сразу при помощи нампая, то стоит делать так.
```suggestion
    arr1000_7 = np.arange(1000, 0, -7)

```